packages/contentstack-import/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@contentstack/cli-cm-import",
   "description": "Contentstack CLI plugin to import content into stack",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
@abhinav-from-contentstack This version is already bumped from 1.3.0 to 1.3.1

Reply:
done